SPECS/libxml2/libxml2.signatures.json

```diff
@@ -1,5 +1,5 @@
 {
   "Signatures": {
-    "libxml2-v2.11.5.tar.gz": "9d9c1026c818c1dadc81d54cc2839cd69c632590f3dcd9b2dc9ef4a136295af1"
```

Comment:
How confident are you in this version bump?  What are the possible compatibility concerns?

Reply:
I have tried to build some dependent packages. Postgres, mariadb, php worked.
But llvm did fail, still looking into it, but looks like for different reason.

I have checked the Docs, there are some new APIs and deprecations. How do we verify if it's safe to version bump?
the 2.11.x won't be supported for long, latest is on 2.13.x

Reply:
Was there resolution on llvm?  Deprecation of API's is usually a sign we shouldn't take the upgrade...we have no idea what the downstream implications will be